File: js/characterMatch.js
exports.isCharacterMatch = function(string1, string2) {

    string1 = string1.toLowerCase().split('')
    string2 = string2.toLowerCase().split('')
    array1 = string1.sort(function (a,b){
        if (a > b)
            return -1;
        if (a < b)
            return 1;
        return 0;
        })
    array2 = string2.sort(function (a,b){
        if (a > b)
            return -1;
        if (a < b)
            return 1;
        return 0;
        })
    for(let i = 0; i < array1.length; i++){
        if(array1[i] != array2[i]){
            return false
        }
    }
    return true
};

File: python/character_match.py
# Don't forget to run your tests!

from ntpath import join


def is_character_match(string1, string2):
	string_1_list = list(string1.lower())
	string_1_list.sort()
	string_1 = ''.join(string_1_list)

	string_2_list = list(string2.lower())
	string_2_list.sort()
	string_2 = ''.join(string_2_list)

	return(True if string_1 == string_2 else False)
	# Your code here

# print(is_character_match('Dennis','DIENNS'))